From bb8801fb21ac8d73e3b1a11a8555aaf822182a51 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 5 Sep 2017 10:36:52 -0700
Subject: [PATCH] libau: Define STRIP weakly

STRIP can be set on environment to use cross version of strip utility
hardcoding to strip is not working in cross environment

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
Upstream-Status: Submitted

 libau/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/libau/Makefile b/libau/Makefile
index 81520ac..c25eef2 100644
--- a/libau/Makefile
+++ b/libau/Makefile
@@ -25,6 +25,7 @@ LibSoHdr = libau.h rdu.h
 LibSoExport = $(addsuffix .exp, $(basename ${LibSoObj}))
 LibSoExportErr = $(addsuffix .err, ${LibSoExport})
 LibSoVerScript = verscript
+STRIP ?= strip
 
 all: ${LibSo}
 
@@ -77,7 +78,7 @@ ${LibSo}.${LibSoMajor}.${LibSoMinor}: ${LibSoObj}
 	${CC} --shared -Wl,-soname,${LibSo}.${LibSoMajor} \
 		-Wl,--version-script,${LibSoVerScript} \
 		${LDFLAGS} -o $@ $^ ${LDLIBS}
-	strip -R EXP $@
+	${STRIP} -R EXP $@
 #	readelf --syms --use-dynamic libau.so
 
 install_ulib: File = ${LibSo}.${LibSoMajor}.${LibSoMinor}
-- 
2.14.1

